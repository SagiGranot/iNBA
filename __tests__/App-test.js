import 'react-native'
import React from 'react'
import App from '../App'
import renderer from 'react-test-renderer'
import Leaders from '../components/leaders'
describe('App snapshot', () => {
  it('renders the loading screen', async () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders the root without loading screen', async () => {
    const tree = renderer.create(<App skipLoadingScreen />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('<App />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree.children.length).toBe(1)
  })
})

describe('TodoList tests', function() {
  test('renders correctly', async () => {
    const data = [
      {
        average: {
          minutes: 36.9,
          points: 25.6,
          off_rebounds: 1.09,
          def_rebounds: 3.93,
          rebounds: 5.01
        },
        player: {
          id: 'ff461754-ad20-4eeb-af02-2b46cc980b24',
          full_name: 'Bradley Beal',
          first_name: 'Bradley',
          last_name: 'Beal',
          position: 'G'
        },
        rank: 1,
        score: 3028,
        teams: [
          {
            id: '583ec8d4-fb46-11e1-82cb-f4ce4684ea4c',
            market: 'Washington',
            name: 'Wizards',
            reference: '1610612764'
          }
        ],
        total: {
          games_played: 82,
          games_started: 82,
          minutes: 3028,
          field_goals_made: 764,
          field_goals_att: 1609
        }
      }
    ]

    const testRenderer = renderer.create(<Leaders data={data} tableName="Test" />)
    expect(testRenderer.root.props).toHaveProperty('data')
    expect(testRenderer.root.props.data).toHaveLength(1)
  })
})
