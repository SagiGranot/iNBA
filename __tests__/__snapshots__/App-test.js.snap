// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders the loading screen 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "overflow": "hidden",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={false}
        pointerEvents="auto"
        removeClippedSubviews={false}
        style={
          Array [
            Object {
              "flex": 1,
              "overflow": "hidden",
            },
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "opacity": 1,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            onLayout={[Function]}
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View
              enabled={false}
              hitSlop={
                Object {
                  "right": 50,
                }
              }
              maxDeltaY={20}
              minOffsetX={5}
              onGestureEvent={[Function]}
              onHandlerStateChange={[Function]}
            >
              <View
                style={
                  Object {
                    "backgroundColor": "#eee",
                    "flex": 1,
                    "flexDirection": "column-reverse",
                    "overflow": "hidden",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    pointerEvents="auto"
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "opacity": 1,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                        "transform": Array [
                          Object {
                            "translateX": 0,
                          },
                          Object {
                            "translateY": 0,
                          },
                        ],
                      }
                    }
                  >
                    <View
                      accessibilityElementsHidden={false}
                      style={
                        Object {
                          "backgroundColor": "#fff",
                          "flex": 1,
                        }
                      }
                    >
                      <View
                        accessibilityIgnoresInvertColors={true}
                        style={
                          Object {
                            "height": "100%",
                            "width": "100%",
                          }
                        }
                      >
                        <Image
                          source={1}
                          style={
                            Array [
                              Object {
                                "bottom": 0,
                                "left": 0,
                                "position": "absolute",
                                "right": 0,
                                "top": 0,
                              },
                              Object {
                                "height": "100%",
                                "width": "100%",
                              },
                              undefined,
                            ]
                          }
                        />
                        <View
                          style={
                            Object {
                              "alignSelf": "center",
                              "position": "absolute",
                              "top": 300,
                            }
                          }
                        >
                          <Image
                            source={1}
                          />
                        </View>
                      </View>
                    </View>
                  </View>
                </View>
                <View
                  onLayout={[Function]}
                  pointerEvents="box-none"
                  style={
                    Object {
                      "left": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "backgroundColor": "#FFF",
                        "transform": Array [
                          Object {
                            "translateX": -750,
                          },
                        ],
                      }
                    }
                  >
                    <View
                      onLayout={[Function]}
                      pointerEvents="box-none"
                      style={
                        Object {
                          "backgroundColor": "#FFF",
                          "borderBottomColor": "#A7A7AA",
                          "borderBottomWidth": 0.5,
                          "height": 64,
                          "paddingBottom": 0,
                          "paddingLeft": 0,
                          "paddingRight": 0,
                          "paddingTop": 20,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "flex": 1,
                          }
                        }
                      />
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      onLayout={[Function]}
      pointerEvents="auto"
      style={
        Object {
          "bottom": 0,
          "elevation": 8,
          "left": 0,
          "right": 0,
        }
      }
    >
      <View
        onLayout={[Function]}
        pointerEvents="box-none"
        style={
          Object {
            "backgroundColor": "rgba(0, 0, 0, 0.8)",
            "borderTopColor": "rgba(0, 0, 0, .3)",
            "borderTopWidth": 0.5,
            "flexDirection": "row",
            "height": 50,
            "paddingBottom": 0,
            "paddingLeft": 0,
            "paddingRight": 0,
            "paddingTop": 0,
          }
        }
      >
        <View
          accessibilityLabel="Home, tab, 1 of 4"
          accessibilityRole="button"
          accessibilityStates={
            Array [
              "selected",
            ]
          }
          accessible={true}
          hitSlop={
            Object {
              "bottom": 5,
              "left": 15,
              "right": 15,
              "top": 0,
            }
          }
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
              },
              Object {
                "backgroundColor": "transparent",
              },
              Object {
                "flexDirection": "column",
                "justifyContent": "flex-end",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 29,
                },
                false,
                Object {
                  "flex": 1,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 1,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 0,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
          </View>
          <Text
            allowFontScaling={true}
            numberOfLines={1}
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#007AFF",
                "fontSize": 11,
                "marginBottom": 1.5,
                "textAlign": "center",
              }
            }
          >
            Home
          </Text>
        </View>
        <View
          accessibilityLabel="Predictions, tab, 2 of 4"
          accessibilityRole="button"
          accessibilityStates={Array []}
          accessible={true}
          hitSlop={
            Object {
              "bottom": 5,
              "left": 15,
              "right": 15,
              "top": 0,
            }
          }
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
              },
              Object {
                "backgroundColor": "transparent",
              },
              Object {
                "flexDirection": "column",
                "justifyContent": "flex-end",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 29,
                },
                false,
                Object {
                  "flex": 1,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 0,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 1,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
          </View>
          <Text
            allowFontScaling={true}
            numberOfLines={1}
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#8E8E93",
                "fontSize": 11,
                "marginBottom": 1.5,
                "textAlign": "center",
              }
            }
          >
            Predictions
          </Text>
        </View>
        <View
          accessibilityLabel="Leaders, tab, 3 of 4"
          accessibilityRole="button"
          accessibilityStates={Array []}
          accessible={true}
          hitSlop={
            Object {
              "bottom": 5,
              "left": 15,
              "right": 15,
              "top": 0,
            }
          }
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
              },
              Object {
                "backgroundColor": "transparent",
              },
              Object {
                "flexDirection": "column",
                "justifyContent": "flex-end",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 29,
                },
                false,
                Object {
                  "flex": 1,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 0,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 1,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
          </View>
          <Text
            allowFontScaling={true}
            numberOfLines={1}
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#8E8E93",
                "fontSize": 11,
                "marginBottom": 1.5,
                "textAlign": "center",
              }
            }
          >
            Leaders
          </Text>
        </View>
        <View
          accessibilityLabel="Standings, tab, 4 of 4"
          accessibilityRole="button"
          accessibilityStates={Array []}
          accessible={true}
          hitSlop={
            Object {
              "bottom": 5,
              "left": 15,
              "right": 15,
              "top": 0,
            }
          }
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
              },
              Object {
                "backgroundColor": "transparent",
              },
              Object {
                "flexDirection": "column",
                "justifyContent": "flex-end",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 29,
                },
                false,
                Object {
                  "flex": 1,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 0,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 1,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
          </View>
          <Text
            allowFontScaling={true}
            numberOfLines={1}
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#8E8E93",
                "fontSize": 11,
                "marginBottom": 1.5,
                "textAlign": "center",
              }
            }
          >
            Standings
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`App snapshot renders the root without loading screen 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "overflow": "hidden",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={false}
        pointerEvents="auto"
        removeClippedSubviews={false}
        style={
          Array [
            Object {
              "flex": 1,
              "overflow": "hidden",
            },
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "opacity": 1,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            onLayout={[Function]}
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View
              enabled={false}
              hitSlop={
                Object {
                  "right": 50,
                }
              }
              maxDeltaY={20}
              minOffsetX={5}
              onGestureEvent={[Function]}
              onHandlerStateChange={[Function]}
            >
              <View
                style={
                  Object {
                    "backgroundColor": "#eee",
                    "flex": 1,
                    "flexDirection": "column-reverse",
                    "overflow": "hidden",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    pointerEvents="auto"
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "opacity": 1,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                        "transform": Array [
                          Object {
                            "translateX": 0,
                          },
                          Object {
                            "translateY": 0,
                          },
                        ],
                      }
                    }
                  >
                    <View
                      accessibilityElementsHidden={false}
                      style={
                        Object {
                          "backgroundColor": "#fff",
                          "flex": 1,
                        }
                      }
                    >
                      <View
                        accessibilityIgnoresInvertColors={true}
                        style={
                          Object {
                            "height": "100%",
                            "width": "100%",
                          }
                        }
                      >
                        <Image
                          source={1}
                          style={
                            Array [
                              Object {
                                "bottom": 0,
                                "left": 0,
                                "position": "absolute",
                                "right": 0,
                                "top": 0,
                              },
                              Object {
                                "height": "100%",
                                "width": "100%",
                              },
                              undefined,
                            ]
                          }
                        />
                        <View
                          style={
                            Object {
                              "alignSelf": "center",
                              "position": "absolute",
                              "top": 300,
                            }
                          }
                        >
                          <Image
                            source={1}
                          />
                        </View>
                      </View>
                    </View>
                  </View>
                </View>
                <View
                  onLayout={[Function]}
                  pointerEvents="box-none"
                  style={
                    Object {
                      "left": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "backgroundColor": "#FFF",
                        "transform": Array [
                          Object {
                            "translateX": -750,
                          },
                        ],
                      }
                    }
                  >
                    <View
                      onLayout={[Function]}
                      pointerEvents="box-none"
                      style={
                        Object {
                          "backgroundColor": "#FFF",
                          "borderBottomColor": "#A7A7AA",
                          "borderBottomWidth": 0.5,
                          "height": 64,
                          "paddingBottom": 0,
                          "paddingLeft": 0,
                          "paddingRight": 0,
                          "paddingTop": 20,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "flex": 1,
                          }
                        }
                      />
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      onLayout={[Function]}
      pointerEvents="auto"
      style={
        Object {
          "bottom": 0,
          "elevation": 8,
          "left": 0,
          "right": 0,
        }
      }
    >
      <View
        onLayout={[Function]}
        pointerEvents="box-none"
        style={
          Object {
            "backgroundColor": "rgba(0, 0, 0, 0.8)",
            "borderTopColor": "rgba(0, 0, 0, .3)",
            "borderTopWidth": 0.5,
            "flexDirection": "row",
            "height": 50,
            "paddingBottom": 0,
            "paddingLeft": 0,
            "paddingRight": 0,
            "paddingTop": 0,
          }
        }
      >
        <View
          accessibilityLabel="Home, tab, 1 of 4"
          accessibilityRole="button"
          accessibilityStates={
            Array [
              "selected",
            ]
          }
          accessible={true}
          hitSlop={
            Object {
              "bottom": 5,
              "left": 15,
              "right": 15,
              "top": 0,
            }
          }
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
              },
              Object {
                "backgroundColor": "transparent",
              },
              Object {
                "flexDirection": "column",
                "justifyContent": "flex-end",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 29,
                },
                false,
                Object {
                  "flex": 1,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 1,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 0,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
          </View>
          <Text
            allowFontScaling={true}
            numberOfLines={1}
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#007AFF",
                "fontSize": 11,
                "marginBottom": 1.5,
                "textAlign": "center",
              }
            }
          >
            Home
          </Text>
        </View>
        <View
          accessibilityLabel="Predictions, tab, 2 of 4"
          accessibilityRole="button"
          accessibilityStates={Array []}
          accessible={true}
          hitSlop={
            Object {
              "bottom": 5,
              "left": 15,
              "right": 15,
              "top": 0,
            }
          }
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
              },
              Object {
                "backgroundColor": "transparent",
              },
              Object {
                "flexDirection": "column",
                "justifyContent": "flex-end",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 29,
                },
                false,
                Object {
                  "flex": 1,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 0,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 1,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
          </View>
          <Text
            allowFontScaling={true}
            numberOfLines={1}
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#8E8E93",
                "fontSize": 11,
                "marginBottom": 1.5,
                "textAlign": "center",
              }
            }
          >
            Predictions
          </Text>
        </View>
        <View
          accessibilityLabel="Leaders, tab, 3 of 4"
          accessibilityRole="button"
          accessibilityStates={Array []}
          accessible={true}
          hitSlop={
            Object {
              "bottom": 5,
              "left": 15,
              "right": 15,
              "top": 0,
            }
          }
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
              },
              Object {
                "backgroundColor": "transparent",
              },
              Object {
                "flexDirection": "column",
                "justifyContent": "flex-end",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 29,
                },
                false,
                Object {
                  "flex": 1,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 0,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 1,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
          </View>
          <Text
            allowFontScaling={true}
            numberOfLines={1}
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#8E8E93",
                "fontSize": 11,
                "marginBottom": 1.5,
                "textAlign": "center",
              }
            }
          >
            Leaders
          </Text>
        </View>
        <View
          accessibilityLabel="Standings, tab, 4 of 4"
          accessibilityRole="button"
          accessibilityStates={Array []}
          accessible={true}
          hitSlop={
            Object {
              "bottom": 5,
              "left": 15,
              "right": 15,
              "top": 0,
            }
          }
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
              },
              Object {
                "backgroundColor": "transparent",
              },
              Object {
                "flexDirection": "column",
                "justifyContent": "flex-end",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "height": 29,
                },
                false,
                Object {
                  "flex": 1,
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 0,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "height": "100%",
                  "justifyContent": "center",
                  "minWidth": 25,
                  "opacity": 1,
                  "position": "absolute",
                  "width": "100%",
                }
              }
            >
              <Text />
            </View>
          </View>
          <Text
            allowFontScaling={true}
            numberOfLines={1}
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#8E8E93",
                "fontSize": 11,
                "marginBottom": 1.5,
                "textAlign": "center",
              }
            }
          >
            Standings
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
